test/Microsoft.Extensions.Configuration.FunctionalTests/ConfigurationTests.cs

```diff
@@ -680,9 +680,7 @@ public void CanSetValuesAndReloadValues()
         }
 
 
-        [ConditionalTheory]
-        // Disabling until https://github.com/aspnet/Configuration/issues/628 is resolved.
-        [OSSkipCondition(OperatingSystems.MacOSX)]
+        [ConditionalTheory(Skip = "https://github.com/aspnet/Configuration/issues/628")]       
```

Comment:
Nope, you still need `OSSkipCondition`. The skip is only meant to say why you're skipping the test.